annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -351,6 +350,25 @@ public Annotation disableAnnotation(Annotation annotation) {
 
 	return persistentAnnotation;
     }
+    
+    @Override
+    public Annotation enableAnnotation(AnnotationId annoId) throws AnnotationServiceException {
+		PersistentAnnotation persistentAnnotation;
+		try {	    
+			persistentAnnotation = getMongoPersistence().find(annoId);
+		    persistentAnnotation.setDisabled(null);
+		    persistentAnnotation = getMongoPersistence().update(persistentAnnotation);
+		} catch (Exception e) {
+		    throw new RuntimeException(e);
```

Comment:
must not throw a runtime exception but a proper catched exception

Reply:
Improved.